packages/gatsby-design-tokens/src/transition.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  curve: {
+    default: `cubic-bezier(0.4, 0, 0.2, 1)`,
+  },
+  speed: {
+    default: `250ms`,
+    fast: `100ms`,
+    slow: `350ms`,
```

Comment:
could we add another one 
`slower: 500ms`
we use it a lot on Cloud, I mean 500ms, the name is less important, if you have better name, :+1:

Reply:
Done in 074716c